src/seth/libexec/seth/seth-calldata

```diff
@@ -12,7 +12,8 @@ if (process.argv.length >= 3 &&
   const sig = process.argv[2];
   try {
     const funcs = new ethers.utils.Interface(["function " + sig.replace(')(', ') returns (')]).functions;
-      console.log(funcs[Object.keys(funcs)[0]].encode(process.argv.slice(3)))
+    const args = process.argv.slice(3).map(JSON.parse)
```

Comment:
This seems like it could fail, because [`JSON.parse`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse) takes an additional optional argument, and `Array.map` passes 3 args to the callback -  cf. [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), section *Tricky use case*.

Better be explicit IMO:

```suggestion
    const args = process.argv.slice(3).map(arg => JSON.parse(arg))
```